packages/rum-core/src/error-logging/error-logging.js

```diff
@@ -48,32 +48,34 @@ class ErrorLogging {
       culprit = lastFrame.filename
     }
 
-    const message =
-      errorEvent.message || (errorEvent.error && errorEvent.error.message)
```

Comment:
The logic here prioritises `errorEvent.message` over `errorEvent.error.message` if they both exists! The change reverses this priority, is that intentional? What's the reasoning?

Reply:
I didn't get you here. 

https://github.com/elastic/apm-agent-rum-js/blob/cd916b88395b879b6b472b8733d63fbe2938ef90/packages/rum-core/src/error-logging/error-logging.js#L52-L59

We set it to `errorEvent.message` first at line 52 and set it to `errorEvent.error.message` later on line 56.

Reply:
Consider this:

```
let errorEvent = {message:'message', error: {message:'error-message'}}
```

The previous logic will result in `errorMessage` to be `message` since `errorEvent.message || (errorEvent.error && errorEvent.error.message)` only checks the first part of the OR operator and never checks the `error`. While the current changes will set `errorMessage` to `error-message`

Reply:
Got it now, It was intentional that my impression that both messages are similar, I was wrong here. I will update and also add test case for the same.